packages/neuron-wallet/package.json

```diff
@@ -18,8 +18,8 @@
   "license": "MIT",
   "scripts": {
     "start": "electron .",
-    "start:dev": "yarn run build && electron .",
-    "build": "ttsc && ncp ./src/block-sync-renderer/index.html ./dist/block-sync-renderer/index.html",
+    "start:dev": "ttsc && electron .",
```

Comment:
> This modification is unnecessary.

Because we no longer need `. /src/block-sync-renderer/index.html` file, we still need to remove the copy html command. But we do can leave the "start:dev" command unchanged.

Reply:
Yes, I mean we can keep `yarn build` in `start:dev`